mev_inspect/classifiers/specs/balancer.py

```diff
@@ -6,15 +9,25 @@
     ClassifierSpec,
     SwapClassifier,
 )
-
+from mev_inspect.classifiers.utils import create_swap_from_transfers
```

Comment:
"utils" is a placeholder word

how about `mev_inspect.classifiers.swaps`?

Reply:
Thanks, great feedback